server/views/account/reset-progress.jade

```diff
@@ -0,0 +1,25 @@
+extends ../layout
+block content
+    include ../partials/flyer
+    #modal-dialog.modal.animated.wobble
+        .modal-dialog
+            .modal-content
```

Comment:
[![https://gyazo.com/ba901b0f18c386912c82aeedba5ca9e8](https://i.gyazo.com/ba901b0f18c386912c82aeedba5ca9e8.png)](https://gyazo.com/ba901b0f18c386912c82aeedba5ca9e8)

Should this be moved into the center of the screen?
It defiantly shouldn't be overlapping the header.